docs/containers/local-process-kubernetes.md

```diff
@@ -0,0 +1,178 @@
+
+Local Process with Kubernetes avoids having to build and deploy your code to your cluster by instead creating a connection directly between your development computer and your cluster. Connecting your development computer to your cluster while debugging allows you to quickly test and develop your service in the context of the full application without creating any Docker or Kubernetes configuration.
+
+Local Process with Kubernetes redirects traffic between your connected Kubernetes cluster and your development computer. This traffic redirection allows code on your development computer and services running in your Kubernetes cluster to communicate as if they are in the same Kubernetes cluster. Local Process with Kubernetes also provides a way to replicate environment variables and mounted volumes available to pods in your Kubernetes cluster in your development computer. Providing access to environment variables and mounted volumes on your development computer allows you to quickly work on your code without having to replicate those dependencies manually.
+
+In this guide, you will learn how to use Local Process with Kubernetes to redirect traffic between your Kubernetes cluster and code running on your development computer. This guide also provides a script for deploying a large sample application with multiple microservices on a Kubernetes cluster.
+
+> **Important**: This feature is currently in preview. Previews are made available to you on the condition that you agree to the [supplemental terms of use][preview-terms]. Some aspects of this feature may change prior to general availability (GA).
+
+## Before you begin
+
+This guide uses the [Bike Sharing sample application][bike-sharing-github] to demonstrate connecting your development computer to a Kubernetes cluster. If you already have your own application running on a Kubernetes cluster, you can still follow the steps below and use the names of your own services.
+
+### Prerequisites
+
+* An Azure subscription. If you don't have an Azure subscription, you can create a [free account](https://azure.microsoft.com/free).
+* [Azure CLI installed][azure-cli].
+* [Visual Studio Code][vs-code] running on macOS or Windows 10.
+* The [Azure Dev Spaces][azds-vs-code] extension version 2.0.220200601 or greater installed in Visual Studio Code.
+* [Azure Dev Spaces CLI installed][azds-cli].
+* A Kubernetes cluster. If you don't have one, use Azure Kubernetes Service to create one by following this [quickstart][azure-kubernetes-service].
```

Comment:
I'd kinda like to avoid sending them to another quickstart.  
Could we give them the script?

az group create --name MyResourceGroup --location eastus
az aks create --resource-group MyResourceGroup --name MyAKS \
    --location eastus --node-count 3 --generate-ssh-keys

Reply:
Yeah, sorry @ghogen. I think we should leave the AZ pre-req and still guide customers to creating the resource group and cluster in our docs

Reply:
Restored the cluster create instructions that we had before